sdk/storage/inc/blobs/blob_client.hpp

```diff
@@ -357,13 +357,36 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief Breaks the previously-acquired lease.
      *
-     * @param options Optional
-     * parameters to execute this function.
+     * @param options Optional parameters to execute this function.
      * @return A BrokenLease describing the broken lease.
      */
     Azure::Core::Response<BrokenLease> BreakLease(
         const BreakBlobLeaseOptions& options = BreakBlobLeaseOptions()) const;
 
+    /**
+     * @brief Sets tags on the underlying blob.
+     * 
+     * @param tags The tags to set
+     * on the blob.
+     * @param options Optional parameters to execute this function.
+     *
+     * @return A SetBlobTagsInfo on successfully setting tags.
+     */
+    Azure::Core::Response<SetBlobTagsInfo> SetTags(
+        Tags tags,
+        const SetBlobTagsOptions& options = SetBlobTagsOptions()) const;
+
+    /**
+     * @brief Gets the tags associated with the underlying blob.
+     * 
+     * @param
+     * options Optional parameters to execute this function.
+     * @return Tags on successfully
+     * getting tags.
+     */
```

Comment:
Please fix the format.
To be more specific, any comment should not be split into multiple lines before the collum limit is hit. Also, each new line should be approximately aligned in all multiple-line comments.